sympy/solvers/solveset.py

```diff
@@ -871,3 +872,162 @@ def solveset(f, symbol=None):
             return solveset_real(f, symbol)
         else:
             return solveset_complex(f, symbol)
+
+
+###############################################################################
+################################ LINSOLVE #####################################
+###############################################################################
+
+
+def eq_to_matrix(equations, *symbols):
```

Comment:
Though the name was inspired from [Matlab](http://in.mathworks.com/help/symbolic/equationstomatrix.html).